mlir/lib/Transforms/Utils/LoopUtils.cpp

```diff
@@ -1653,6 +1653,28 @@ void mlir::scalarReplace(AffineForOp forOp) {
   LLVM_DEBUG(llvm::dbgs() << "SCAL REP END\n");
 }
 
+// Replace affine.for with a 1-d affine.parallel and clone the former's body
+// into the latter while remapping values.
+void mlir::affineParallelize(AffineForOp forOp) {
+  Location loc = forOp.getLoc();
+  OpBuilder outsideBuilder(forOp);
+  // Creating empty 1-D affine.parallel op
+  auto newPloop = outsideBuilder.create<AffineParallelOp>(
+      loc, forOp.getLowerBoundMap(), forOp.getLowerBoundOperands(),
+      forOp.getUpperBoundMap(), forOp.getUpperBoundOperands());
+  auto forOpIV = forOp.getInductionVar();
+  auto parallelIV = newPloop.getIVs()[0];
+  BlockAndValueMapping operandMap;
+  operandMap.map(forOpIV, parallelIV);
+  // Setting insertion point to start of affine parallel op
+  OpBuilder ob = OpBuilder::atBlockBegin(newPloop.getBody());
+  // Remapping arguments from affine.for op to affine.parallel op
+  for (auto &op : forOp.getBody()->without_terminator()) {
+    ob.clone(op, operandMap);
+  }
```

Comment:
Drop trivial braces (you can drop whenever you have a single statement in the `for`).